models/item.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var item = sequelize.define('item', {
+    name: DataTypes.STRING
+  }, {
+    classMethods: {
+      item.associate: function(models) {
```

Comment:
Since this should be outside of the parenthesis, the association statement should look like `item.associate = (models) => {` Here we are using a lambda instead of function.